docs/indyav/onboarding.rst

```diff
@@ -0,0 +1,26 @@
+Getting Started on Indy AV
+==========================
+
+Collaboration
+-------------
+
+Slack
+*****
+Join the slack to keep updated on the developement at `uf-cimar <uf-cimar.slack.com>`_.
+Any valid @ufl.edu email will be able to create an account.
+
+Github
+******
+If you see this documentation you are probably already aware of the repository
+
+Grabcad
+*******
+We use GrabCAD Workbench to manage mechanical design files.
+
+Software
+--------
+We will follow MIL's `current development practices <https://github.com/uf-mil/mil/blob/master/docs/development/development_guide.md>`_
```

Comment:
``We will follow MIL's `current development practices <https://github.com/uf-mil/mil/blob/master/docs/development/development_guide.md>`_`` -> ``We will follow `MIL's development practices <../development/development_guide.html>`_.``